### PR TITLE
Update Kubernetes from 1.18.4 to 1.18.8

### PR DESCRIPTION
**Pull request by briandowns**
#### Proposed Changes ####

Update references to Kubernetes 1.18.4 to 1.18.8.

#### Linked Issues ####

rancher/rke2#255

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -69,15 +69,15 @@ RUN echo ${CACHEBUST}>/dev/null
 RUN CHART_VERSION="v3.13.3"     CHART_FILE=/charts/rke2-canal-chart.yml             CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.10.101"    CHART_FILE=/charts/rke2-coredns-chart.yml           CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.36.300"    CHART_FILE=/charts/rke2-ingress-nginx-chart.yml     CHART_BOOTSTRAP=false   /charts/build-chart.sh
-RUN CHART_VERSION="v1.18.4"     CHART_FILE=/charts/rke2-kube-proxy-chart.yml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
+RUN CHART_VERSION="v1.18.8"     CHART_FILE=/charts/rke2-kube-proxy-chart.yml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
```
Comment (dweomer) [inline Dockerfile:72]: This should be causing the build to fail as the chart asset specified by the version doesn ot exist. See https://github.com/rancher/rke2-charts/tree/master/assets/rke2-kube-proxy

Comment (dweomer) [reply Dockerfile:72]: It's because my charts/build-chart.sh script is garbage

Comment (dweomer) [reply Dockerfile:72]: This failed correctly:
```shell
#!/usr/bin/env bash
set -eux -o pipefail
: "${CHART_FILE?required}"
: "${CHART_NAME:="$(basename "${CHART_FILE%%-chart.yml}")"}"
: "${CHART_URL:="${CHART_REPO:="https://rke2-charts.rancher.io"}/assets/${CHART_NAME}/${CHART_NAME}-${CHART_VERSION:="v0.0.0"}.tgz"}"
curl -fsSL "${CHART_URL}" -o "${CHART_TMP:=$(mktemp)}"
cat <<-EOF > "${CHART_FILE}"
apiVersion: helm.cattle.io/v1
kind: HelmChart
metadata:
  name: "${CHART_NAME}"
  namespace: "${CHART_NAMESPACE:="kube-system"}"
  annotations:
    helm.cattle.io/chart-url: "${CHART_URL}"
spec:
  bootstrap: ${CHART_BOOTSTRAP:=false}
  chartContent: $(base64 -w0 < "${CHART_TMP}")
EOF
```

Comment (briandowns) [reply Dockerfile:72]: Updated in latest commit.

**Review by brandond — APPROVED** [review]

**Review by dweomer — APPROVED** [review]

**Pull request merged**